api/user/schema.py

```diff
@@ -118,11 +119,46 @@ def mutate(self, info, email, **kwargs):
         exact_user = active_user.filter(UserModel.email == email).first()
         if not exact_user:
             raise GraphQLError("User not found")
-        user_role = UsersRole.query.filter_by(user_id=exact_user.id).first()
-        new_role = kwargs.pop('role_id')
-        user_role.role_id = new_role
-        with SaveContextManager(user_role, 'Role id', new_role):
-            return ChangeUserRole(user=exact_user)
+
+        if not exact_user.roles:
```

Comment:
Has this line been tested? Might be the reason the coverage is failing.

Reply:
@bevkololi  thanx for pointing this out